test/integration-test/tests/smoke.rs

```diff
@@ -1,14 +1,19 @@
+use procfs::KernelVersion;
+
 use aya::{
     include_bytes_aligned,
     programs::{Extension, Xdp, XdpFlags},
     Bpf, BpfLoader,
 };
 
-mod common;
-use common::kernel_version;
-
 #[test]
 fn xdp() {
+    let kernel_version = KernelVersion::current().unwrap();
+    if kernel_version < KernelVersion::new(5, 18, 0) {
```

Comment:
Let's maybe address that in another PR, but it would be nice to have some basic XDP test which can run on kernels older than 5.18.0.